src/rotate.c

```diff
@@ -0,0 +1,38 @@
+#include "rotate.h"
+
+void rotate_block(Block* b, const int rotation_matrix[2][2]){
+    /* Rotates block 90 degrees
+    using matrix multiplication with rotation matrix.*/
+    int matC[4][2]={0};
```

Comment:
can you change matC to something more descriptive. Also afaik `= {0}` is not necessary here